src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -236,10 +235,12 @@ export const ProjectOverview: FC = () => {
             <Grid item>
               <DataButton
                 loading={!projectOverviewData}
-                secured={projectOverviewData?.project.location}
+                secured={projectOverviewData?.project.primaryLocation}
                 empty="Enter Location"
                 redacted="You do not have permission to view location"
-                children={displayLocation}
+                children={
+                  projectOverviewData?.project.primaryLocation.value?.name.value
+                }
```

Comment:
We'll want to display the field region here as well, but this can wait until we do the update functionality.